scrapi/util.py

```diff
@@ -36,3 +36,7 @@ def stamp_from_raw(raw_doc, **kwargs):
 
 def format_date_with_slashes(date):
     return date.strftime('%m/%d/%Y')
+
+
+def create_rename_iterable(documents, source, target, dry):
```

Comment:
What is this for? If it is for a specific migration, I'd prefer it stay in the migrations file.